librz/core/cmd_info.c

```diff
@@ -1101,84 +1101,44 @@ RZ_IPI int rz_cmd_info(void *data, const char *input) {
 				if (!obj) {
 					break;
 				}
-				if (input[2] && input[2] != '*' && input[2] != 'j' && !strstr(input, "qq")) {
-					bool rizin = strstr(input, "**") != NULL;
+				bool rizin = input[1] == ' ' && strstr(input, "**");
```

Comment:
@ret2libc please take a look at these changes since you work on the `i` commands refactoring.

Reply:
i had to fix this code, because it was not working properly also before.